### DISCUSSION TITLE
LDAP client for Security Onion Console

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#1533] [Ideas] [answered]
I wish I could use LDAP to log in to Security Onion Console.

**Comment by shipler** [comment, marked as answer]
I wish such feature would be in the platform as well, but from what stated in the comments here - https://www.reddit.com/r/securityonion/comments/gniprc/security_onion_hybrid_hunter_130_beta_2_available/
SO are not in the direction of integrating the platform with domain features.

> Once an attacker gets a foothold in your environment, they will most likely try to compromise ldap/AD, so we typically recommend against integrating ANY security device into ldap/AD.

**Reply by Masaya-A** [reply]
Thank you for your reply. I missed the comment.
